src/PrestaShopBundle/Form/Admin/Sell/Product/Image/ImageDropzoneType.php

```diff
@@ -76,6 +76,11 @@ public function configureOptions(OptionsResolver $resolver)
                 'window.delete' => $this->trans('Delete selection', 'Admin.Actions'),
                 'window.close' => $this->trans('Close window', 'Admin.Actions'),
                 'window.zoom' => $this->trans('Zoom on selection', 'Admin.Actions'),
+                'modal.close' => $this->trans('Close', 'Admin.Actions'),
+                'modal.accept' => $this->trans('Accept', 'Admin.Actions'),
+                'modal.message' => $this->trans('Test of message', 'Admin.Actions'),
+                'modal.title' => $this->trans('Are you sure about deleting %filesNb% file(s)?', 'Admin.Actions'),
```

Comment:
```suggestion
                'modal.title' => $this->transChoice('Are you sure you want to delete the selected image?|Are you sure you want to delete the %filesNb% selected images?', 'Admin.Catalog.Notification'),
```
@jolelievre told me about this solution to manage plurals. See: https://symfony.com/doc/3.4/translation.html#pluralization